cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/module/lib/inmemory/InMemoryTableModule.java

```diff
@@ -31,6 +30,5 @@ public void register(DatasetDefinitionRegistry registry) {
     registry.add(new InMemoryTableDefinition("table"));
     registry.add(new InMemoryTableDefinition(Table.class.getName()));
     registry.add(new InMemoryTableDefinition("orderedTable"));
```

Comment:
That I am not sure. Any suggestions? I would love it get rid of it, just not sure if there is way to "deprecate" it first?